crates/uv/src/commands/venv.rs

```diff
@@ -188,5 +189,22 @@ async fn venv_impl(
         }
     }
 
+    match &platform.os() {
+        Os::Windows => {
```

Comment:
Note we usually use a `cfg` for this, e.g. (from elsewhere)

```rust
    let bin_name = if cfg!(unix) {
        "bin"
    } else if cfg!(windows) {
        "Scripts"
    } else {
        unimplemented!("Only Windows and Unix are supported")
    };
```

Reply:
Do we want to include `unimplemented!("Only Windows and Unix are supported")` as well for consistency?